.github/workflows/preview-env-deploy.yml

```
@@ -28,9 +28,28 @@ jobs:
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_BUCKET_NAME;
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_GCLOUD_SA_KEY;
 
+      - name: Shorten commit SHA
+        run: echo "short_sha=$(echo ${{ github.sha }} | cut -c1-8)" >> $GITHUB_ENV
```

Short commit by GitHub standards is 7 chars not 8 ([ref](https://support.configura.com/hc/en-us/articles/360052644053-Are-Git-Commit-id-s-Truncated-in-CET-Developer#:~:text=The%20conclusion%20is%20that%20git,to%20always%20use%208%20characters.)):

```suggestion
        run: echo "short_sha=$(echo ${{ github.sha }} | cut -c1-7)" >> $GITHUB_ENV
```

---

Actually, you also don't need to be doing the short SHA calculation at all. GitHub will shorten it automatically.

For example, when you mention in markdown
```
cfd70262bef268b3a615decc553e2208c18b29f5
```

it looks like this in the UI: cfd70262bef268b3a615decc553e2208c18b29f5